albumentations/augmentations/transforms.py

```diff
@@ -332,32 +336,70 @@ class ImageCompression(ImageOnlyTransform):
     """
 
     class InitSchema(BaseTransformInitSchema):
-        quality_lower: int = Field(default=99, description="Lower bound on the image quality", ge=1, le=100)
-        quality_upper: int = Field(default=100, description="Upper bound on the image quality", ge=1, le=100)
+        _size_len = 2
+        quality_range: Optional[Union[Tuple[int, int], List[int]]] = Field(
+            default=(99, 100),
+            description="lower and upper bound on the image quality as tuple (lower_bound, upper_bound)",
+        )
+        quality_lower: Optional[int] = Field(
+            default=None,
+            description="Lower bound on the image quality",
+            ge=1,
+            le=100,
+        )
+        quality_upper: Optional[int] = Field(
+            default=None,
+            description="Upper bound on the image quality",
+            ge=1,
+            le=100,
+        )
         compression_type: ImageCompressionType = Field(
             default=ImageCompressionType.JPEG,
             description="Image compression format",
         )
 
         @model_validator(mode="after")
-        def validate_quality(self) -> Self:
-            if self.quality_lower >= self.quality_upper:
-                msg = "quality_lower must be less than quality_upper"
-                raise ValueError(msg)
+        def validate_ranges(self) -> Self:
+            lower = 1
+            upper = 100
+
+            def validate_range(range: Tuple[int, int], name: str) -> None:
```

Comment:
Pydantic should do this validation for us.

When we say in `InitShema`:

`quality_range: Optional[Tuple[int, int]]`

It will throw an error if anything but None of Tuple[int, int]will be used => You can delete:

```
if len(range) != self._size_len:
                    msg = f"'quality_range' should have lenght of {self._size_len} but lenght of {len(range)} was found"
                    raise ValueError(msg)
```